connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -777,6 +783,30 @@ private void startTask(ConnectorTaskId taskId) {
         worker.startTask(taskId, taskConfig, connConfig, this, initialState);
     }
 
+    private Callable<Void> getTaskCallable(final ConnectorTaskId taskId, final TaskStatus.State action) {
+        return new Callable<Void>() {
+            @Override
+            public Void call() throws Exception {
+                switch (action) {
+                    case UNASSIGNED:
```

Comment:
I pondered between `UNASSIGNED`, `PAUSED` or both of them. Would you suggest `PAUSED`?  To be clear, this doesn't examine the current state of the task. It's just an indication which action to take (start or stop a task) and it's explicitly specified by the caller of the method. 

If a new enum was used, this case would have been called `START`. 
Will change the argument name.